internal/services/machinelearning/machine_learning_workspace_resource_test.go

```diff
@@ -355,6 +374,18 @@ func (r WorkspaceResource) Exists(ctx context.Context, client *clients.Client, s
 func (r WorkspaceResource) basic(data acceptance.TestData) string {
 	template := r.template(data)
 	return fmt.Sprintf(`
+provider "azurerm" {
+  features {
+    key_vault {
+      purge_soft_delete_on_destroy       = false
+      purge_soft_deleted_keys_on_destroy = false
+    }
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
```

Comment:
Is there a reason we need to add these to all of the tests?

Reply:
Hi @stephybun, this was already in the tests but has been pulled into every test to get around copy pasting the template. This is needed because the workspace is soft deleted which will make the test fail as resources are still kicking around in the resource group